index.md

```diff
@@ -1,4 +1,7 @@
 ---
 layout: default
 title: Home
----
\ No newline at end of file
+---
+
+{% include about.html %}
```

Comment:
Let's put all the include statements in a layout html file and not in a `.md` file. `.md` files are only for content. Make a separate layout file extending default and include the htmls there and then use that as layout instead of the default.